copis/gl/glcanvas.py

```diff
@@ -362,7 +372,6 @@ def on_idle(self, event: wx.IdleEvent) -> None:
 
     def on_key(self, event: wx.KeyEvent) -> None:
         """Handle EVT_KEY_DOWN and EVT_KEY_UP."""
```

Comment:
Can I have my `pass` statements back? Otherwise the integrity of our program hinges on the existence of a docstring. Which is very disconcerting because it _technically_ works. Also in `on_erase_background` and `_render_cameras`.

Reply:
Woo